autopush/web/registration.py

```diff
@@ -59,20 +59,10 @@ def extract_data(self, req):
         chid = req['path_kwargs'].get('chid', router_data.get("channelID"))
         if uaid:
             try:
-                u_uuid = uuid.UUID(uaid)
+                uuid.UUID(uaid)
             except (ValueError, TypeError):
                 raise InvalidRequest("Invalid Request UAID",
                                      status_code=401, errno=109)
-            # Check if the UAID has a 'critical error' which means that it's
-            # probably invalid and should be reset/re-registered
-            try:
-                record = self.context['settings'].router.get_uaid(u_uuid.hex)
-                if record.get('critical_failure'):
```

Comment:
If it's a critical failure, what is the appropriate behavior? I'm worried about the reasoning for these changes being lost since it happened purely in an IRC chat afaik.